src/prelaunch/outpost.a

```diff
@@ -0,0 +1,20 @@
+;license:MIT
+;(c) 2019 by qkumba
```

Comment:
Give yourself credit please

Reply:
Thank you. If the file was from an existing cheat I will make it qkumba or 4am/Frank M. Can go back and change the others too, if you want.

Reply:
Sure, in a separate pull request.